mlptrain/configurations/configuration_set.py

```diff
@@ -664,23 +664,37 @@ def _load_npz(self, filename: str) -> None:
                 mult=int(data['M'][i]),
                 box=None if box.has_zero_volume else box,
             )
-
-            if data['R_plumed'].ndim > 0:
-                config.plumed_coordinates = np.array(
-                    data['R_plumed'][i], dtype=float
+            try:
```

Comment:
Just a note that this is fixing an issue unrelated to autoDE, where the code could not load npz files that were created before the Plumed integration was merged. Might be nice to document this problem in an issue and link it to this PR.